docs/requirements.txt

```diff
@@ -6,6 +6,7 @@ sphinx-favicon
 sphinx-gallery
 sphinx_autodoc_typehints==1.12.0
 myst-nb
+napari[dev]
```

Comment:
With https://github.com/napari/napari/pull/5438 I wonder if this is sufficient—in my experience the viewer does open, so I think it does need Qt, while now `napari[dev]` doesn't auto-install a Qt backend.
So for the live 0.4.17 behavior, using pyqt5, the default, this should be `napari[pyqt,dev]`, as per:
https://github.com/napari/docs/pull/78

Reply:
I agree.

Reply:
Wait, the just edited in this PR contribution stuff already says to install dev napari explicitly:
```
- a development installation of napari. Follow the [contributor guide](dev-installation) for details on how to do this;
```
**That** guide explains how [dev] works.

So I think we should just remove this line, because if you leave as is, then no QT backend, so stuff won't work. If you make it `[pyqt,dev]` then it only works when pypi wheel of pyqt is available—not on my mac for example.

Reply:
Did we use these requirements in any place? If no then maybe just remove it and add `docs` extra to the napari setup.cfg?

Reply:
I'd be pro on a napari[docs], as I noted above:
https://github.com/napari/docs/pull/41#issuecomment-1372710387
Basically, [dev] installs [testing] which is pretty heavy. For docs I think we just need the examples to build.

Reply:
100%, `napari[docs]` is the way

Reply:
So maybe for now we remove it here, so this can be merged, then make napari PR and then make a new PR here to adjust the copy and the requirements.txt?

Reply:
I can open a follow up as soon as this is merged. Let me remove this line here then